### ISSUE TITLE
Android Studio only downloads the first sources JAR for a SNAPSHOT dependency

### ISSUE THREAD

**Issue by sixbangs** [jitpack/jitpack.io#757]
Using the following dependency definition, the JAR and the associated sources JAR get downloaded properly.

``` groovy
compile('org.bitbucket.my-org:repository-name:develop-SNAPSHOT') {
    changing = true;
}
```

When I push a new commit and do a Gradle sync, the latest JAR is downloaded properly, as expected - but the new sources don't get downloaded. I don't get the new sources even if I delete the cached source JAR for the old SNAPSHOT. Is this an Android Studio issue or a JitPack issue? Is there a workaround other than depending on specific releases/commit hashes?

File > Synchronize and Gradle sync don't resolve the problem.

**Comment by jitpack-io** [comment]
Hi @6bangs ,

It could be related to this bug:  https://code.google.com/p/android/issues/detail?id=188831
Since it works the first time this is most likely a bug in the IDE.

**Comment by 6bangs** [comment]
Hmm... maybe related, but I don't think it's exactly the same thing. In my case, the new SNAPSHOT gets downloaded, but not the new source jar. With regard to that bug, does the situation change at all if you add `changing = true`?

**Comment by jitpack-io** [comment]
True, but also all artifacts with `-SNAPSHOT` versions are treated as `changing`.

**Comment by jamesliao333** [comment]
Hi, I got the same problem, did you solve it?

**Comment by jamesliao333** [comment]
O(∩_∩)O~您好，您的邮件我已收到，我会及时查看的了。谢谢！！

**Comment by jamesliao333** [comment]
O(∩_∩)O~您好，您的邮件我已收到，我会及时查看的了。谢谢！！
